pkgs/tools/misc/complete-alias/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "complete-alias";
+  version = "1.10.0";
+
+  src = fetchFromGitHub {
+    owner = "cykerway";
+    repo = pname;
+    rev = version;
+    sha256 = "1s0prdnmb2qnzc8d7ddldzqa53yc10qq0lbgx2l9dzmz8pdwylyc";
+  };
+
+  buildPhase = ''
+    # required for the patchShebangs setup hook
+    chmod a+x complete_alias
```

Comment:
This is the same as +x, right?

Reply:
Yes, that is correct I'm using it most of the time to clearly indicate what is happening.

But it can be changed if you want, no problem at all.